INSTALL

```diff
@@ -2,7 +2,7 @@
  --------------------
 
  This document describes installation on all supported operating
- systems (the Linux/Unix family, OpenVMS and Windows)
+ systems (the Linux/Unix family including Mac OS/X, OpenVMS and Windows)
```

Comment:
This makes it sound like the Linux/Unix family includes Mac OS/X, OpenVMS and Windows!

Reply:
Agreed.
Perhaps better wording "systems (Linux, Unix, *BSD, Mac OS/X, OpenVMS, Windows)"
OR 
"systems (Linux, Mac OS/X, OpenVMS, Windows, and Unix derivatives)"